Tests/test_SeqIO_write.py

```diff
@@ -76,7 +76,7 @@
 
 
 class WriterTests(unittest.TestCase):
-    """Cunning unit test where methods are added at run time."""
+    """Cunning unit test where methods are added at run time."""  # TODO - Let's not be cunning
```

Comment:
Why? The only obvious alternative is a *lot* of boilerplate code defining test methods for example example file.

Reply:
There must be a better way than what we have now. Currently when a test fails, it is unclear which test failed.